backend/chunker/chunker.go

```diff
@@ -405,6 +427,20 @@ func (f *Fs) setHashType(hashType string) error {
 	return nil
 }
 
+func (f *Fs) setTransactionMode(transactionMode string) error {
+	switch transactionMode {
+	case "rename":
+		f.useRename = true
+	case "norename":
+		f.useRename = false
+	case "auto":
+		f.useRename = f.CanQuickRename()
+	default:
+		return fmt.Errorf("unsupported transaction mode '%s'", transactionMode)
+	}
+	return nil
```

Comment:
Tracking transaction-marked chunks is obviously impossible without metadata.
setTransactionMode should refuse `useRename==true` when `metaFormat==none`.
returning `errors.New("incompatible transaction options")` or alike

Reply:
The rclone integration test includes `nometa` case https://github.com/rclone/rclone/blob/v1.53.3/fstest/test_all/config.yaml#L42,  so you'll have to tweak (`t.Skip("...")` conditionally) the new test case https://github.com/rclone/rclone/pull/4675#issuecomment-748709047.